### ISSUE TITLE
Hunters Vale Major Lootbag Bug?

### ISSUE THREAD

**Issue by Kab00sh** [WarEmu/WarBugs#19456]
i just finished hunters vale on my shaman and got a goldbag.
this is all it contained. shouldnt be intended,

![issue](https://user-images.githubusercontent.com/52345008/134415543-4d2630b5-5b32-42e7-944d-d3e8cc67ed0d.PNG)
 right?

**Comment by lilimx** [comment]
Hey,

what do you mean is the issue here?

**Comment by Kab00sh** [comment]
shouldnt there be a trinket in there?

**Comment by lilimx** [comment]
Hey,

what trinket? From the set?

**Comment by Kab00sh** [comment]
i was told this should be the case, yes

**Comment by lilimx** [comment]
Hey,

the ring is the last piece in the quest chain
where you have to get kill on every boss in HV in under 45min.

**Comment by Tesq** [comment]
he mean event itiem, purple

**Comment by SilverWF** [comment]
He mean event slot item.
Just got gold bag at my shaman and it has pretty the same items inside as at his screenshot, but instead of horn I has cape.
No event slot item too.

**Comment by Kab00sh** [comment]
Yes, i knew something was suppost to be in there, but it looks like it isnt anymore or atleast not 100% of the time. Is this an intended change or a bug?

**Comment by Tesq** [comment]
the event itiem slot dosent appear all the time the bags yep
